geometry_msgs/Pose object_pos
float32 color